aries-site/src/layouts/content/InPageNavigation.js

```diff
@@ -0,0 +1,126 @@
+import { useState, useRef, useEffect } from 'react';
+import Link from 'next/link';
+import PropTypes from 'prop-types';
+import { Box, Button, Nav, Text } from 'grommet';
+import { siteContents } from '../../data/search/contentForSearch';
+import { nameToSlug } from '../../utils';
+
+const useActiveHeadingId = (headings, options) => {
+  const [activeHeadingId, setActiveHeadingId] = useState();
+  const observer = useRef();
+  // useEffect finds which page heading is on screen as the user scrolls
+  useEffect(() => {
+    // if at top of page, do not show active heading on ToC
+    window.onscroll = () => {
+      if (window.pageYOffset === 0) {
+        setActiveHeadingId(null);
+      } else {
+        // find page active page heading
+        const elements = headings.map(heading =>
+          document.getElementById(nameToSlug(heading[1])),
+        );
+        observer.current?.disconnect();
+        observer.current = new IntersectionObserver(entries => {
+          entries.forEach(entry => {
+            if (entry?.isIntersecting) {
+              setActiveHeadingId(entry.target.id);
+            }
+          });
+        }, options);
+        elements.forEach(el => {
+          if (el) {
+            observer.current?.observe(el);
+          }
+        });
+      }
+      return () => observer.current?.disconnect();
+    };
+  }, [headings, options]);
+  return activeHeadingId;
+};
+
+export const InPageNavigation = ({ title }) => {
+  const match = siteContents.find(
+    item => item.name.toLowerCase() === title.toLowerCase(),
+  );
+
+  const regexp = new RegExp(/#{1,} (...+?) ?~{2}/, 'g');
+  const headings = match && [...match.content.matchAll(regexp)];
+
+  // top and bottom margin values to calculate intersection window
+  const topMargin = '50';
+  const bottomMargin = '-95';
+  const activeId = useActiveHeadingId(headings, {
+    rootMargin: `${topMargin}% 0% ${bottomMargin}% 0%`,
+  });
+
+  return headings.length > 0 ? (
+    <Box
+      pad={{ horizontal: 'xsmall' }} // pad for keyboard focus
+      style={{
+        height: 'calc(100vh - 50px)',
+        overflowY: 'auto',
+        position: 'sticky',
+        marginTop: '50px',
```

Comment:
I think that "Jump to section" should be vertically aligned with page title to start.

<img width="1677" alt="Screen Shot 2022-08-02 at 11 02 35 AM" src="https://user-images.githubusercontent.com/12522275/182443119-0234672a-d0cf-4716-b812-41f09d4cf1c4.png">